lib/auth/github.go

```diff
@@ -274,6 +274,8 @@ func populateGithubClaims(client githubAPIClientI) (*services.GithubClaims, erro
 	if err != nil {
 		return nil, trace.Wrap(err, "failed to query Github user teams")
 	}
+	log.Debugf("Retrieved %v teams for GitHub user %v", len(teams), user.Login)
```

Comment:
missing dot. Also you can use structured logging to list teams:

```go
log.WithFields(log.Fields{"teams:" teams, "user": user.Login})
```

Reply:
The full list of teams is printed a few lines below. I added this line as a quick sanity check when we are debugging things for customers. For example maybe pagination breaks at some point in the future, this is a good diagnostic line to tell at a glance something is wrong.